### ISSUE TITLE
Android redirection broken after upgrade

### ISSUE THREAD

**Issue by camchis** [auth0/react-native-auth0#632]
### Checklist

- [X] The issue can be reproduced in the [react-native-auth0 sample app](https://github.com/auth0-samples/auth0-react-native-sample) (or N/A).
- [X] I have looked into the [Readme](https://github.com/auth0/react-native-auth0#readme), [Examples](https://github.com/auth0/react-native-auth0/blob/master/EXAMPLES.md), and [FAQ](https://github.com/auth0/react-native-auth0/blob/master/FAQ.md) and have not found a suitable solution or answer.
- [X] I have looked into the [API documentation](https://auth0.github.io/react-native-auth0/) and have not found a suitable solution or answer.
- [X] I have searched the [issues](https://github.com/auth0/react-native-auth0/issues) and have not found a suitable solution or answer.
- [X] I have searched the [Auth0 Community](https://community.auth0.com) forums and have not found a suitable solution or answer.
- [X] I agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Description

After updating our React Native version and many other packages including `react-native-auth0`, redirection after logging in successfully seems to be broken on Android (no problems on iOS)

We have applied the breaking changes that were introduced in version `2.9.0` and still seem to be having problems

build.gradle
`manifestPlaceholders = [auth0Domain: "xxx.x.com", auth0Scheme: "com.token.auth0"]`

AndroidManifest
```
      <activity
        android:name="MainActivity"
        android:label="@string/app_name"
        android:screenOrientation="portrait"
        android:theme="@style/AppTheme"
        android:exported="true"
        android:windowSoftInputMode="adjustPan">
        <intent-filter>
          <action android:name="android.intent.action.MAIN" />
          <category android:name="android.intent.category.LAUNCHER" />
        </intent-filter>
        <intent-filter>
          <action android:name="android.intent.action.VIEW" />
          <category android:name="android.intent.category.DEFAULT" />
          <category android:name="android.intent.category.BROWSABLE" />
          <data
            android:scheme="com.token.auth0" />
        </intent-filter>
        <intent-filter>
            <action android:name="android.intent.action.VIEW" />
            <category android:name="android.intent.category.BROWSABLE"/>
            <data android:scheme="https"/>
        </intent-filter>
      </activity>
```

And we are including the `customScheme` when we invoke the authorize method
```
      const { accessToken, refreshToken, idToken, expiresIn, scope } = await this.webAuth.authorize({
        audience: x,
        scope: x,
        connection: x,
      }, {
        customScheme: 'com.token.auth0',
      })
```

Result of running `adb shell dumpsys package r`:

```
      com.token.auth0:
        30b904c com.token.android/com.auth0.react.RedirectActivity
```

- Not an expo project
- `customScheme` does not clash with `applicationId`
- Added callback to allowed callback URLs in dashboard

After successful login, the page just hangs and the user is not redirected back to the app, this happens on Android only

Everything was working fine pre-upgrade (RN 0.64.4, react-native-auth0 2.8.3)

### Reproduction

1. Log in successfully on Android
2. No redirection back to app

### react-native-auth0 version

2.14.1

### React Native version

0.70.7

### Platform

Android

### Platform version(s)

Any

**Comment by jdicami** [comment]
@camchis why did you close, what was the solution, I am experiencing the same issue

**Comment by felipem292** [comment]
@jdicami did you solved this?, i having the same problem

**Comment by poovamraj** [comment]
@jdicami @felipem292 Is this after upgrading to v3? Have you ensure that you have followed the [migration guide](https://github.com/auth0/react-native-auth0/blob/master/MIGRATION_GUIDE.md) especially the part on migrating callback URL?

**Comment by hamza-busuri** [comment]
@poovamraj Experiencing this issue aswell. Why was this issue closed? We upgraded from to 2.17.4 and RN version 0.71.13. I've passed in a customScheme to the authorize call and I have the manifestPlaceholders. Any ideas what the problem could be?
